mw-webapp/src/component/link/Link.tsx

```diff
@@ -22,17 +22,27 @@ interface LinkProps {
    * Data attribute for cypress testing
    */
   dataCy?: string;
+
+  /**
+   * Open link in a new tab or window
+   * @default isNewTab: false
+   */
+  isNewTab?: boolean;
 }
 
 /**
  * Link component
  */
 export const Link = (props: PropsWithChildren<LinkProps>) => {
+  const isNewTab = props.isNewTab ?? false;
```

Comment:
it's unneeded wrapper. props.isNewTab is already boolean value

Reply:
fixed